### DISCUSSION TITLE
Initial account balance cannot be 0 sat ?

### DISCUSSION THREAD

**Discussion by jimmydjabali** [lightninglabs/lightning-terminal#716] [Q&A] [answered]
Hello,

I'd like to create empty accounts with litd but seems the minimum to create an account is 1 sat.
Trying with CLI, I get this error:
`initial balance cannot be smaller than 1`
https://github.com/lightninglabs/lightning-terminal/blob/c03b040e4de02c813d7613f580df0e6f7ae5f111/cmd/litcli/accounts.go#L117-L119

Trying with RPC, I get this error:
`a new account cannot have balance of 0`
https://github.com/lightninglabs/lightning-terminal/blob/master/accounts/store.go#L110-L112

Is this is a technical limitation why it's not possible to create an account with an initial balance of 0 sat, or is there a workaround to make it possible?

Thanks a lot!

**Comment by levmi** [comment, marked as answer]
Previously, this wasn't possible. But, thanks to your question, we just merged a pull request to make this possible in the next release of litd: https://github.com/lightninglabs/lightning-terminal/pull/718 Let me know if you any questions!
